packages/theme/designColors.json

```diff
@@ -0,0 +1,1546 @@
+{
+    "light": {
+        "background": {
+            "surface": {
+                "elevation (-1)": {
+                    "description": "",
+                    "type": "color",
+                    "value": "#eeeeeeff",
+                    "blendMode": "normal",
+                    "extensions": {
+                        "org.lukasoppermann.figmaDesignTokens": {
+                            "styleId": "S:3eb1242659c39f8bef9e8e855d31362300ee1541,",
+                            "exportKey": "color"
+                        }
+                    }
```

Comment:
nit: Is there better way to export this from figma? We're only using value and it seems useless to store large JSON just to get to 1 property out of 5

Reply:
Unfortunately not. After discussion with designers, this is the leanest version of color config they can get.